_posts/2018-04-27-scrum-vs-kanban.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+authors: [nils_devos]
+title: "Scrum VS Kanban"
+image: /img/agile/scrum-vs-kanban/boxers.jpg
+tags: [Agile,Scrum, Kanban]
+category: Agile
+comments: true
+permalink: agile/2018/04/27/scrum-vs-kanban.html
+---
+
+# Scrum VS Kanban
+
+> Scrum and Kanban are both agile frameworks that are used to deliver software, or any other sort of project for that matter, in incremental and iterative way. 
+Both terms have become incorrectly interchangeable. Although there are many similarities between the two, 
+understanding the key differences between both Scrum and Kanban will bring clarity. 
+
+## The basics
+As with all agile frameworks, we have a product owner, a development team and an intermediate person, 
+known in Scrum as the Scrum Master and in Kanban he or she is known as the Agile Coach. 
+
+ All three work together to get most value to a client in the shortest amount of time.  Work is divided into stories and those stories are put on a product backlog. So far, so good, both Scrum and Kanban do the same at the very basic level. The way both handle these stories is where the difference lies. 
+
+## Scrum
+
+A Scrum team works in Sprints, a Time-boxed period in which the development team delivers a new iteration of the product.
+Most commonly these Sprints last 2 weeks, although 3 or even 4 weeks Sprint aren’t a rare sight. 
+
+Each Sprint start with a Sprint planning meeting. 
+In this meeting the development team, Scrum Master and Product Owner decide which stories the development team will tackle during this Sprint. 
+These selected stories are known as the Sprint Backlog. This backlog should not change during the Sprint. 
+
+On a daily basis in the Sprint, there is a Daily Standup meeting, a 15-minute standup meeting in which every member of the development team answers 3 simple questions: 
+'What have I done yesterday', 'What will I do today' and 'Are there any problems that block me in going forward with my task'.
+
+After a Sprint there are 2 rituals; the Sprint review meeting, where all stakeholders are invited to a demonstration of new functionality added to the product;
```

Comment:
And "Two" here.

Reply:
... of new functionalit**ies** added ...
or 
... of **a** new functionality added ...